File: python/03.10.2018/sumacyfr.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  sumacyfr.py

def sumuj_cyfry1(liczba):
    suma = 0
    while liczba > 0:
        suma += liczba % 10
        liczba = liczba // 10
    return suma
def main(args):
    liczba = int(input("Podaj liczbę 2-cyfrową: "))
    print("Suma: ", sumuj_cyfry(liczba))
        
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: C++/sumacyfr/sumacyfr.cpp
/*
 * sumacyfr.cpp
 */


#include <iostream>

using namespace std;

int sumuj_cyfry1(int liczba){
    int suma = 0;
    while(liczba > 0){
        suma += liczba % 10;
        liczba = int(liczba / 10);
    }
    return suma;
}

int main(int argc, char **argv)
{
    int liczba = 0;
    while(liczba < 10){
        cout << "Podaj liczbę 2-cyfrową:";
        cin >> liczba;
    }
    int x = sumuj_cyfry1(liczba);
    cout << x;
	return 0;
}

